cfme/tests/containers/test_labels.py

```diff
@@ -31,9 +31,9 @@
 
 
 def check_labels_in_ui(instance, name, expected_value):
-    if hasattr(instance.summary, 'labels') and \
-            hasattr(instance.summary.labels, name.lower()):
-        return getattr(instance.summary.labels, name.lower()).text_value == str(expected_value)
+    view = refresh_and_navigate(instance, 'Details')
+    if view.entities.labels.is_displayed:
+        return view.entities.labels.read().get(name) == str(expected_value)
```

Comment:
Optional: I would advice you to replace such code with get_text_of.
Ex:
`
net_name = view.entities.properties.get_text_of('Name')
`

Reply:
Make sense :smile: